mantidproject/mantid
# Test the doc tests from mediawiki examples nightly

Fixes #13355.

**To test**:
- Check the Jenkins job: http://builds.mantidproject.org/view/All/job/master_doctests_wikitests. The build has been working well for a few days.
- Initially we had one failure in one of the SliceViewer examples (outdated expected output). Currently all tests pass.
- Check that it is actually testing the special mediawiki tests (the ones `mwTests/mwTest_...`). There should be >=1004 tests in total.
- I think the current location of TestWikiPython.py is fine. An alternative could be to move it to `docs/` where you also find the scripts that build and test the documentation.
- As discussed in #13311, the "mwTests" take less than a minute to generate and ~2 minutes to run, so it should be safe to have them included in this job.

Note: when this is merged, the job configuration needs to be updated to execute the modified `buildconfig/Jenkins/doctests` instead of the temporary shell text.
